addon/components/frost-button.js

```diff
@@ -1,14 +1,11 @@
+/**
+ * Component definition for the frost-button component
+ */
 import Ember from 'ember'
-const {
-  Component,
-  get,
-  isEmpty,
-  Logger,
-  typeOf,
-  ViewUtils
-} = Ember
+const {Component, Logger, ViewUtils, get, isEmpty, typeOf} = Ember
```

Comment:
I personally prefer multiple lines for clarity. I think we need to establish a standard here, because I might be wrong.